libpod/container_internal_linux.go

```diff
@@ -2225,7 +2225,7 @@ func (c *Container) getOCICgroupPath() (string, error) {
 	cgroupManager := c.CgroupManager()
 	switch {
 	case (rootless.IsRootless() && (cgroupManager == config.CgroupfsCgroupsManager || !unified)) || c.config.NoCgroups:
-		return "", nil
+		return c.config.CgroupParent, nil
```

Comment:
Do we risk breaking existing rootless containers that have CgroupParent set to the default?

Reply:
good point.

Not sure what is the best way to deal with it.  Should we stop setting it if it is the default value (perhaps until podman 4.x)?

Reply:
Is there any way we can do an access check on the cgroup, see if we have the ability to use it without error? If not, then your suggestion is definitely the best way forward.

Reply:
I think that will be error prone and difficult to get right.  We won't only have to check if the current user has permissions to the cgroup, but that any ID mapped in the user namespace has it.  And that will be even more complicated with cgroup v1.

Looking more into it, I don't think it will be a problem if we just check for `c.config.CgroupParent == CgroupfsDefaultCgroupParent` as that cgroup is usually owned by root.

What do you think about the new version?

Reply:
Alright, this SGTM